src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/coreperson/CorePersonEventListener.kt

```diff
@@ -31,15 +31,28 @@ class CorePersonEventListener(
           val eventType = sqsMessage.MessageAttributes!!.eventType.Value
           if (eventFeatureSwitch.isEnabled(eventType, "coreperson")) {
             when (eventType) {
-//              "OFFENDER_INSERTED" ->service.offenderAdded(sqsMessage.Message.fromJson())
-//              "OFFENDER_UPDATED" -> service.offenderUpdated(sqsMessage.Message.fromJson())
-//              "OFFENDER_DELETED" -> service.offenderDeleted(sqsMessage.Message.fromJson())
-//              "OFFENDER_IDENTIFIER-INSERTED" ->service.offenderIdentifierAdded(sqsMessage.Message.fromJson())
+              "OFFENDER-INSERTED" -> service.offenderAdded(sqsMessage.Message.fromJson())
+              "OFFENDER-UPDATED" -> service.offenderUpdated(sqsMessage.Message.fromJson())
+              "OFFENDER-DELETED" -> service.offenderDeleted(sqsMessage.Message.fromJson())
+
+              "OFFENDER_BOOKING-INSERTED" -> service.offenderBookingAdded(sqsMessage.Message.fromJson())
+              "OFFENDER_BOOKING-CHANGED" -> service.offenderBookingUpdated(sqsMessage.Message.fromJson())
+              "OFFENDER_BOOKING-REASSIGNED" -> service.offenderBookingReassigned(sqsMessage.Message.fromJson())
+
+              "OFFENDER_SENTENCES-INSERTED" -> service.offenderSentenceAdded(sqsMessage.Message.fromJson())
+              "OFFENDER_SENTENCES-UPDATED" -> service.offenderSentenceUpdated(sqsMessage.Message.fromJson())
+              "OFFENDER_SENTENCES-DELETED" -> service.offenderSentenceDeleted(sqsMessage.Message.fromJson())
```

Comment:
I'm interested to see how many of the updated events are raised, some of the offence related events have xtag code to only raise if there is an actual change as there were so many being raised